src/main/scala/com/fulcrumgenomics/sopt/Sopt.scala

```diff
@@ -56,7 +58,25 @@ object Sopt {
                          hidden: Boolean,
                          description: String,
                          args: Seq[Arg]
-                        ) extends MarkDownDescription
+                        ) extends MarkDownDescription {
+    /** Returns a command line string representation of the arguments.  If user-specified values are set on the args,
+      * it will use those, otherwise the defaults if present, otherwise it the type of the argument.
```

Comment:
last part after the comma doesn't read well.  Also, why would it output the type if there is no user set value or default value?  I would say they should be omitted if no value is present.